src/dev/code_coverage/ingest_coverage/__tests__/enumerate_patterns.test.js

```diff
@@ -17,21 +17,27 @@
  * under the License.
  */
 
-import { createFailError } from '@kbn/dev-utils';
-import { ES_HOST } from '../constants';
-import { pretty, green } from '../utils';
+import expect from '@kbn/expect';
+import { enumeratePatterns } from '../team_assignment/enumerate_patterns';
+import { resolve } from 'path';
+import { ToolingLog } from '@kbn/dev-utils';
 
-const { Client } = require('@elastic/elasticsearch');
+const ROOT = resolve(__dirname, '../../../../..');
```

Comment:
```suggestion
import { REPO_ROOT } from '@kbn/dev-utils';
```

Reply:
It should blow up

Reply:
Does this repo root "find" the correct root?  If so, that's bomber!

Reply:
>Does this repo root "find" the correct root? If so, that's bomber!

yes, it will find the correct one on its own :)

Reply:
Thanks for the tip man!